src/entities/FormattingNode/types/InlineToolData.ts

```diff
@@ -0,0 +1,16 @@
+import { create, Nominal } from '../../../utils/Nominal';
+
+/**
+ * Base type for Inline Tool data
+ */
+type InlineToolDataBase = Record<string, unknown>;
+
+/**
+ * Nominal type for Inline Tool data
+ */
+export type InlineToolData = Nominal<InlineToolDataBase, 'InlineToolData'>;
+
+/**
+ * Function to cast values to InlineToolData type
+ */
+export const createInlineToolData = create<InlineToolData>();
```

Comment:
it is not a type, so it shouldn't be in a `/types/` folder, i guess.

Reply:
It's not a type, but its a function to cast types. I think it's better to be here